src/main/scala/dpla/ingestion3/model/DplaMapData.scala

```diff
@@ -95,10 +106,15 @@ case class OreAggregation(
                            originalRecord: ExactlyOne[String], //map v4 specifies this as a ref, but that's LDP maybe?
                            hasView: ZeroToMany[EdmWebResource] = Seq(),
                            intermediateProvider: ZeroToOne[EdmAgent] = None,
-                           `object`: ZeroToOne[EdmWebResource] = None, //full item page
-                           preview: ZeroToOne[EdmWebResource] = None, ///thumbnail
+                           `object`: ZeroToOne[EdmWebResource] = None, // full size image (should *not* be used)
```

Comment:
I'm still not sure I can keep this straight in my head. I don't think there is a ticket to do the modeling review (next sprint?). I'll do that tomorrow morning and tag this for that review.